cmd/flags.go

```diff
@@ -245,11 +245,17 @@ func BoolEnv(k string) bool {
 	return b
 }
 
-func EnvOrDefault(key string, defaultVal string) string {
-	if envVal := os.Getenv(key); envVal != "" {
-		return envVal
+func EnvOrDefault(key string, defaultVal string, predicates ...func(string) bool) string {
```

Comment:
❤️ Love the predicate addition. fn programing 🤌

Reply:
Unfortunately this code is no longer needed, but I share your sentiment!